svea_msgs/VehicleState state
---
bool ok